src/templates/ssr/fabric/index.svelte

```diff
@@ -0,0 +1,119 @@
+	import AdvertisementLabel from '$templates/components/AdvertisementLabel.svelte';
+	import Pixel from '$templates/components/Pixel.svelte';
+
+	export let Trackingpixel: GAMVariable;
+	export let Researchpixel: GAMVariable;
+	export let Viewabilitypixel: GAMVariable;
+	export let ClickthroughUrl: GAMVariable;
+	export let thirdPartyJSTracking: GAMVariable;
+
+	export let Layer1BackgroundImage: GAMVariable;
+	export let Layer1BackgroundPosition: GAMVariable;
+	export let Layer2BackgroundImage: GAMVariable;
+	export let Layer2BackgroundPosition: GAMVariable;
+	export let Layer3BackgroundImage: GAMVariable;
+	export let Layer3BackgroundPosition: GAMVariable;
+
+	export let MobileLayer1BackgroundImage: GAMVariable;
+	export let MobileLayer1BackgroundPosition: GAMVariable;
+	export let MobileLayer2BackgroundImage: GAMVariable;
+	export let MobileLayer2BackgroundPosition: GAMVariable;
+	export let MobileLayer3BackgroundImage: GAMVariable;
+	export let MobileLayer3BackgroundPosition: GAMVariable;
+</script>
+
+<AdvertisementLabel />
+<div class="creative--fabric">
+	<a
+		class="gs-container link"
+		href={`${CLICK_MACRO}${ClickthroughUrl}`}
+		target="_blank"
+	>
+		<div class="creative-container">
+			<div
+				class="layer layer1"
+				style:--tablet-background-image={`url('${Layer1BackgroundImage}')`}
+				style:--tablet-background-position={Layer1BackgroundPosition}
+				style:--mobile-background-image={`url('${MobileLayer1BackgroundImage}')`}
+				style:--mobile-background-position={`url('${MobileLayer1BackgroundPosition}')`}
+			/>
+			<div
+				id="layer2"
+				class="layer layer2"
+				style:--tablet-background-image={`url('${Layer2BackgroundImage}')`}
+				style:--tablet-background-position={Layer2BackgroundPosition}
+				style:--mobile-background-image={`url('${MobileLayer2BackgroundImage}')`}
+				style:--mobile-background-position={`url('${MobileLayer2BackgroundPosition}')`}
+			/>
+			<div
+				class="layer layer3"
+				style:--tablet-background-image={`url('${Layer3BackgroundImage}')`}
+				style:--tablet-background-position={Layer3BackgroundPosition}
+				style:--mobile-background-image={`url('${MobileLayer3BackgroundImage}')`}
+				style:--mobile-background-position={`url('${MobileLayer3BackgroundPosition}')`}
+			/>
+		</div>
+	</a>
+	{#if isValidReplacedVariable(Trackingpixel)}
+		<Pixel src={Trackingpixel} />
+	{:else if isValidReplacedVariable(Researchpixel)}
+		<Pixel src={Researchpixel} />
+	{:else if isValidReplacedVariable(Viewabilitypixel)}
+		<Pixel src={Viewabilitypixel} />
+	{/if}
+</div>
+
+{thirdPartyJSTracking}
+
+<style lang="scss">
+	@import '../../components/scss/helpers.scss';
+	.link {
+		display: block;
+		text-decoration: none;
+	}
+	.creative-container {
+		position: relative;
+	}
+	.creative--fabric {
+		display: block;
+		position: relative;
+		&,
+		.link {
+			height: 250px;
+			overflow: hidden;
+		}
+		.layer1 {
+			z-index: 1;
+		}
+		.layer2 {
+			z-index: 2;
+		}
+		.layer3 {
+			z-index: 3;
+		}
```

Comment:
Sorry 🙈, I keep seeing optimisations! This css and the classes are actually redundant as [they'll automatically be stacked](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_without_z-index)

Reply:
Yes you are totally right!